app/src/main/java/com/github/se/polyfit/viewmodel/post/CreatePostViewModel.kt

```diff
@@ -60,8 +63,18 @@ constructor(
     _post.createdAt = createdAt
   }
 
-  fun setPostLocation(location: Location) {
-    _post.location = location
+  fun setPostLocation(): Job {
```

Comment:
It is unusual to have a setter without argument, maybe updatePostLocation or initPostLocation

Reply:
That's true, I just didn't change the current name when I modified it, but I can refactor in initPostLocation